iptables/iptables.go

```diff
@@ -263,6 +277,29 @@ func (ipt *IPTables) Stats(table, chain string) ([][]string, error) {
 	return rows, nil
 }
 
+// StructuredStats returns statistics as structured data which may be further
+// parsed and marshaled.
+func (ipt *IPTables) StructuredStats(table, chain string) ([]Stat, error) {
```

Comment:
I suggest you add a separate `ParseStats` function that just takes a `[]string` and parses the stats accordingly.

Reply:
I originally was going to do exactly that. The only issue I foresee is that exposing a `ParseStats` method that accepts an arbitrary `[]string` (or `[][]string`) may be subject to misuse.

Anyway, two questions I'm looking for your feedback on:

1) I can make a `ParseStat(stat []string) (Stat, error)`  and/or a `ParseStats(stats [][]string) ([]Stat, error)`. Any preference for either or both?

2) Would you like me to remove `StructuredStats(table, chain string) ([]Stat, error)` completely?

Reply:
BTW, I pushed up a contribution with `ParseStat(stat []string) (Stat, error)` and left `StructuredStats(table, chain string) ([]Stat, error)` for now pending feedback.